.codeql-prebuild-cpp-Windows.sh

```diff
@@ -11,18 +11,20 @@ pacman --noconfirm -S \
   gcc \
   git \
   make \
-  mingw-w64-x86_64-binutils \
-  mingw-w64-x86_64-boost \
-  mingw-w64-x86_64-cmake \
-  mingw-w64-x86_64-curl \
-  mingw-w64-x86_64-miniupnpc \
-  mingw-w64-x86_64-nlohmann-json \
-  mingw-w64-x86_64-nodejs \
-  mingw-w64-x86_64-onevpl \
-  mingw-w64-x86_64-openssl \
-  mingw-w64-x86_64-opus \
-  mingw-w64-x86_64-rust \
-  mingw-w64-x86_64-toolchain
+  mingw-w64-ucrt-x86_64-boost \
```

Comment:
Do you think ucrt will work if the setup step isn't using that? https://github.com/LizardByte/Sunshine/blob/7702e55476fef7edf7e421a3631dcde2686afe73/.github/workflows/codeql.yml#L142

If not, I might have to figure out how to handle this separately as we use the same workflow file across all repos. We could also change the other repo using msys2 to ucrt (and we should anyway).

Reply:
Actually the other one is already using ucrt for the ci build, so I will convert codeql to use it as well.

https://github.com/LizardByte/libdisplaydevice/blob/ff361a83c4b2d97df25c33efba99125f2d04b3de/.github/workflows/ci.yml#L58

Reply:
Can you apply these changes directly to this PR?

https://github.com/LizardByte/Sunshine/pull/2581/files

Reply:
I made the updates here: https://github.com/LizardByte/Sunshine/pull/2581

Reply:
does `msystem: ucrt64` need to be added to https://github.com/LizardByte/Sunshine/blob/7702e55476fef7edf7e421a3631dcde2686afe73/.github/workflows/codeql.yml#L146 as well?

Reply:
@tez011 I just merged that, so you should be good if you rebase.